crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -634,13 +635,15 @@ fn create_supported_operations(path: &Path) -> Result<Message, ConversionError>
 
         Ok(Message::new(
             &Topic::new_unchecked(&stopic),
-            Operations::try_new(path)?.create_smartrest_ops_message()?,
+            Operations::try_new(path, Duration::from_secs(0), Duration::from_secs(0))?
```

Comment:
What does it mean to have a timeout of 0 seconds?

Reply:
These tests do not need any timeout options, but the API needs those parameters. I can make them optional, but this will make the code more complicated. So, I went this way.